.github/workflows/note.yml

```diff
@@ -22,14 +22,18 @@ on:
         description: 'The number of hours an AOD PR should be expired since last updated.'
         type: 'number'
         required: true
+      aod_instruction_link:
+        description: 'The link to your AOD instructions.'
+        type: 'string'
+        required: true
 
 env:
   AOD_NOTE: >
     ⛔️ <strong>This is an AOD request, and merging is NOT allowed.</strong>
     The request will be automatically applied once the code owners approve it.
     Please close the PR once you are finished or it will automatically be closed
     after ~${{ inputs.expiry_hours }} hours of it's last update. For more
-    instructions, please see [here](https://github.com/abcxyz/aod-template/blob/main/README.md).
+    instructions, please see [here](${{ inputs.aod_instruction_link }}).
```

Comment:
Could you default it to the repo root README? This should be discoverable.

Reply:
ack

Reply:
defaulted to repo root instead just in case readme is not added, at repo root can also see readme.

Reply:
closing, feel free to reopen.